cfgov/unprocessed/css/forms.less

```diff
@@ -606,29 +608,46 @@
     margin-bottom: unit(30px / @base-font-size-px, em);
 }
 
-.form-l_col-1-2 {
-    .respond-to-min(1060px, {
-        .grid_column(6);
-    });
+.form-label-header + .form-l .form-label-header {
+    // Override heading-5() settings.
+    .webfont-regular();
+    text-transform: none;
+    margin-bottom: unit(10px / @font-size, em);
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/c0a6079f9dce39fd915edd2a890a0eaf94471b69